src/services/avatars.ts

```diff
@@ -0,0 +1,166 @@
+import { Service } from "../service";
+
+interface assoc {
+    [key: string]: any;
+}
+
+export class Avatars extends Service {
+
+    /**
+     * Get Browser Icon
+     *
+     * You can use this endpoint to show different browser icons to your users.
+     * The code argument receives the browser code as it appears in your user
+     * /account/sessions endpoint. Use width, height and quality arguments to
+     * change the output settings.
+     *
+     * @param string code
+     * @param number width
+     * @param number height
+     * @param number quality
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getBrowser(code: string, width: number = 100, height: number = 100, quality: number = 100): Promise<string> {
+        let path = '/avatars/browsers/{code}'.replace(new RegExp('{code}', 'g'), code);
```

Comment:
You can use template literals here
```suggestion
        const path = `/avatars/browsers/${code}`;
```